### DISCUSSION TITLE
OBS中不显示弹幕

### DISCUSSION THREAD

**Discussion by Durrinkun** [xfgryujk/blivechat#242] [提问]
使用本地和公用服务器生成的URL，在OBS中都不显示弹幕，用测试房间的URL也不显示。
更新到1.9.2之后才出现，之前用的是很老的版本，可以正常使用。
OBS版本30.2.3，浏览器源插件使用的https://github.com/obsproject/obs-browser

**Comment by xfgryujk** [comment]
没有复现
